pkg/providers/docker/reconciler/reconciler.go

```diff
@@ -7,7 +7,9 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	anywherev1 "github.com/aws/eks-anywhere/pkg/api/v1alpha1"
+	"github.com/aws/eks-anywhere/pkg/clients/kubernetes"
 	"github.com/aws/eks-anywhere/pkg/cluster"
+	c "github.com/aws/eks-anywhere/pkg/cluster"
```

Comment:
could we not alias this? if there is a conflict with a `cluster` variable, just rename that variable to `c` (or something else)